python/katana/local/_graph.pyx

```diff
@@ -457,22 +460,72 @@ cdef class GraphBase:
 
         :rtype: list[EntityType]
         """
-        cdef const EntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
+        cdef const CEntityTypeManager* manager = &self.underlying_property_graph().GetNodeTypeManager()
         type_ids = manager.GetAtomicEntityTypeIDs()
         types = [EntityType.make(manager, type_id) for type_id in type_ids]
         return types
 
+    def get_node_type_manager(self):
+        """
+        :return the node type manager
+        """
+        return EntityTypeManager.make(&self.underlying_property_graph().GetNodeTypeManager())
+
+    def get_type_of_node(self, uint64_t n):
+        """
+        Return the type id of a node
```

Comment:
This isn't returning *the* type of a node. It's returning the most specific type. The node actually has many types as you can tell by the fact that `does_node_have_type` will return true for various different types.

Please change the documentation for this and the analogous edge function.